src/pudl/package_data/eia860/column_maps/generator_solar_existing.csv

```diff
@@ -0,0 +1,42 @@
+capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw
+summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw
+winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw
+generator_operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month
+generator_operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year
+uses_lenses_mirrors,lenses_mirrors,lenses_mirrors,lenses_mirrors,lenses_mirrors,lenses_mirrors,lenses_mirrors,lenses_mirrors,lenses_mirrors,lenses_mirrors,lenses_mirrors
+uses_single_axis_tracking,single_axis_tracking,single_axis_tracking,single_axis_tracking,single_axis_tracking,single_axis_tracking,single_axis_tracking,single_axis_tracking,single_axis_tracking,single_axis_tracking,single_axis_tracking
+uses_dual_axis_tracking,dual_axis_tracking,dual_axis_tracking,dual_axis_tracking,dual_axis_tracking,dual_axis_tracking,dual_axis_tracking,dual_axis_tracking,dual_axis_tracking,dual_axis_tracking,dual_axis_tracking
+uses_fixed_tilt,fixed_tilt,fixed_tilt,fixed_tilt,fixed_tilt,fixed_tilt,fixed_tilt,fixed_tilt,fixed_tilt,fixed_tilt,fixed_tilt
+uses_east_west_fixed_tilt,,,,east_west_fixed_tilt,east_west_fixed_tilt,east_west_fixed_tilt,east_west_fixed_tilt,east_west_fixed_tilt,east_west_fixed_tilt,east_west_fixed_tilt
+uses_parabolic_trough,parabolic_trough,parabolic_trough,parabolic_trough,parabolic_trough,parabolic_trough,parabolic_trough,parabolic_trough,parabolic_trough,parabolic_trough,parabolic_trough
+uses_linear_fresnel,linear_fresnel,linear_fresnel,linear_fresnel,linear_fresnel,linear_fresnel,linear_fresnel,linear_fresnel,linear_fresnel,linear_fresnel,linear_fresnel
+uses_power_tower,power_tower,power_tower,power_tower,power_tower,power_tower,power_tower,power_tower,power_tower,power_tower,power_tower
+uses_dish_engine,dish_engine,dish_engine,dish_engine,dish_engine,dish_engine,dish_engine,dish_engine,dish_engine,dish_engine,dish_engine
+uses_other_solar_technology,other_solar_technology,other_solar_technology,other_solar_technology,other_solar_technology,other_solar_technology,other_solar_technology,other_solar_technology,other_solar_technology,other_solar_technology,other_solar_technology
+uses_azimuth_angle,,,,,azimuth_angle,azimuth_angle,azimuth_angle,azimuth_angle,azimuth_angle,azimuth_angle
```

Comment:
I don't think the `azimuth angle` column is a boolean. Should probably remove the `uses` prefix.

Reply:
great catch!